cursorless-talon/src/spoken_forms.json

```diff
@@ -81,7 +81,7 @@
       "its": "inferPreviousMark",
       "visible": "visible"
     },
-    "simple_scope_modifier": { "every": "every" },
+    "simple_scope_modifier": { "every": "every", "grand": "grand" },
```

Comment:
Is this the internal identifier we want to use? I guess it's fine, but figured I'd raise it as these are really hard to change cc/ @josharian

Reply:
`grandparent`? `grandscope`? `ancestor`?

Reply:
I think I prefer ancestor. If we in the future add ordinals or some other way to get even higher up in the hierarchy grand parent is the wrong term.

Reply:
Ancestor it is

Reply:
Updated